lib/asciidoctor/standoc/inline.rb

```diff
@@ -107,7 +107,7 @@ def stem_parse(text, xml, style)
           xml.stem math, **{ type: "MathML" }
         elsif style == :latexmath
           latexmlmath_input = Unicode2LaTeX::unicode2latex(text).gsub(/'/, '\\').gsub(/\n/, " ")
-          latex = IO.popen('latexmlmath --preload=amsmath  -- -', 'r+', :external_encoding=>'UTF-8') do |io|
+          latex = IO.popen('latexmlmath --preload=amsmath --inputencoding=UTF-8 -- -', 'r+', :external_encoding=>'UTF-8') do |io|
```

Comment:
@CAMOBAP795 in this case we need a check to ensure LaTeXML is 0.8.4+

Reply:
Can we add a set of methods to check dependencies early on in Metanorma processing?